.github/workflows/reusable-pytest.yml

```
@@ -10,12 +10,18 @@ on:
           ["3.8", "3.9", "3.10"]
         description: JSON string containing the list of python versions to test
         type: string
+      fail_fast:
+        required: false
+        default: true
```

```suggestion
        default: false
```
For pytesting different versions, maybe we should always default to not canceling in-progress jobs...